VKProject/Src/Pages/NewsPage.cs

```diff
@@ -0,0 +1,12 @@
+using OpenQA.Selenium;
+using VKProject.Pages.Base;
+
+namespace VKProject.Pages;
+
+public class NewsPage : BasePage
+{
+    private static readonly By MyPageSelector = By.XPath("//li[@id='l_pr']/a");
```

Comment:
better use // instead of /
If there's need it to be always child, so it;s better to use some atribute of a

Reply:
done